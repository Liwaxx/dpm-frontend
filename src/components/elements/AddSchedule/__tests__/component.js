import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../component';

jest.mock('redux-form');
jest.mock('../../../../configs');

describe('Component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Component classes={{}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with message', () => {
    const tree = renderer
      .create(<Component classes={{}} message="tes" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});