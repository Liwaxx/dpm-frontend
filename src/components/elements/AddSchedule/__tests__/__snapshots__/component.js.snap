// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1"
>
  <form>
    <div />
    <button
      className="MuiButtonBase-root-55 MuiButton-root-29 MuiButton-text-31 MuiButton-flat-34"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-30"
      >
        SAVE 
      </span>
      <span
        className="MuiTouchRipple-root-58"
      />
    </button>
  </form>
</div>
`;

exports[`Component renders with message 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1"
>
  <form>
    <div />
    <button
      className="MuiButtonBase-root-55 MuiButton-root-29 MuiButton-text-31 MuiButton-flat-34"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-30"
      >
        SAVE 
      </span>
      <span
        className="MuiTouchRipple-root-58"
      />
    </button>
  </form>
</div>
`;
